concourse/pipeline-release.yml

```
@@ -63,34 +64,31 @@ set_failed_anchor: &set_failed
         BUCKET_PATH: clusters-google/
         BUCKET_NAME: {{tf-bucket-name}}
 #####################################################
-
 resource_types:
-- name: gcs
-  type: docker-image
-  source:
-    repository: frodenas/gcs-resource
-
-- name: terraform
-  type: docker-image
-  source:
-    repository: ljfranklin/terraform-resource
-    tag: 0.11.14
+    - name: terraform
+      type: registry-image
+      source:
+        repository: ljfranklin/terraform-resource
+        tag: 0.11.14
+    - name: gcs
+      type: registry-image
+      source:
+        repository: frodenas/gcs-resource
 
 resources:
 # Image Resources
 - name: centos-gpdb-dev-7
   type: docker-image
```

```suggestion
  type: registry-image
```

we can use registry-image to get a colorful log.
and releng team like registry-image because it has less pull frequency

---

got it.